src/modules/theme/components/header/header-archived-banner.component.html

```diff
@@ -13,10 +13,13 @@
       <span>
         <strong>This innovation was archived on {{ innovation.statusUpdatedAt | date: ("app.date_formats.long_date" | translate) }}.</strong>
       </span>
-      <span>
+      <span *ngIf="isOwner">
```

Comment:
Since the text is the same except for the last part I would have removed the conditional spans and made this like:
`<span> You can continue to update your innovation record when it is archived. These changes can only be viewed by you and your collaborators. <ng-container *ngIf="isOwner">If you want support on your innovation in future, you can submit your record for a needs reassessment.</ng-container></span>`